### PR TITLE
Fix RecyclerView's scroll position reset on config change

### PR DESCRIPTION
**Pull request by selmanon**

### CODE REVIEW

**Comment by jaredsburrows** [comment]
@selmanon Make sure you set your `git config user.name <>` and git config user.email <>`. Your Github account and local email currently not synced.

File: frontend/android/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -12,11 +12,12 @@
       android:label="@string/application_name"
       android:supportsRtl="true"
       android:theme="@style/Theme.SdkSearch"
-      >
+  >
```
Comment (jaredsburrows) [inline frontend/android/src/main/AndroidManifest.xml]: Most likely not necessary.

File: frontend/android/src/main/java/com/jakewharton/sdksearch/ui/MainActivity.kt [hunk 2]
```diff
@@ -192,6 +197,15 @@ class MainActivity : Activity() {
     disposables.clear()
   }
 
+  override fun onConfigurationChanged(newConfig: Configuration?) {
```
Comment (jaredsburrows) [inline frontend/android/src/main/java/com/jakewharton/sdksearch/ui/MainActivity.kt]: I think it is safe to make `newConfig: Configuration?` be `newConfig: Configuration` and remove the `!!.` or please use `?.`.

**Comment by JakeWharton** [comment]
Thanks for the PR, but this creates a fragile mechanism that I do not want to maintain. See https://github.com/JakeWharton/SdkSearch/issues/39#issuecomment-358395744 for how I plan to solve this.

**Pull request closed without merging**